src/main/java/no/nav/veilarbperson/utils/PersonV2DataMapper.java

```diff
@@ -63,6 +63,11 @@ public static PersonNavnV2 navnMapper(List<HentPerson.Navn> personNavn) {
     }
 
     public static Fnr hentFamiliemedlemFnr(HentPerson.Familiemedlem familiemedlem) {
+        if  ("opphoert".equals(familiemedlem.getFolkeregisterpersonstatus().stream()
+                .findFirst().map( HentPerson.Folkeregisterpersonstatus::getForenkletStatus)
+                .orElse(null))) {
+            return null;
+        }
```

Comment:
Dette _kan_ være en liste, men vi har valgt å ikke hente historikk, så det skal være ganske sikkert at det ikke kommer flere felter i listen, derfor mener vi at findFirst() er ok å bruke..

Reply:
Can you use `getFirstElement(familiemedlem.getFolkeregisterpersonstatus())` instead of `familiemedlem.getFolkeregisterpersonstatus().stream().findFirst()..`...

I saw this function is used somewhere else. In any case, maybe some better formatting or something would make this function bit more readable.